Projektarbeit III/Assets/Scripts/Player/PlayerController/IdleState.cs

```diff
@@ -28,6 +32,14 @@ public void UpdateState()
     {
         // Example: Transition to WalkingState if horizontal input is detected
         Vector2 movementInput = movementAction.ReadValue<Vector2>();
+
+        // Apply fall force when the player starts falling
+        if (rb.linearVelocity.y <= 0)
+        {
+            rb.linearVelocity += Vector2.down * fallForce * Time.deltaTime;
+        }
```

Comment:
Ironically this is only called when the player is grounded. The player remains in the JumpingState until they touch the ground. I mean it works but this should be cleaned up.